Xamarin.Forms.Platform.Android/Renderers/ScrollViewRenderer.cs

```diff
@@ -32,8 +33,9 @@ public class ScrollViewRenderer : NestedScrollView, IVisualElementRenderer, IEff
 		LayoutDirection _prevLayoutDirection = LayoutDirection.Ltr;
 		bool _checkedForRtlScroll = false;
 
-		public ScrollViewRenderer(Context context) : base(context)
+		public ScrollViewRenderer(Context context) : base(new ContextThemeWrapper(context, Resource.Style.NestedScrollBarStyle))
```

Comment:
I think #8090 (which first added this wrapper) caused https://github.com/xamarin/Xamarin.Forms/issues/9103. Waiting for confirmation from customers that reverting that PR resolved the issue. If that's the case, then hopefully we're not reintroducing that problem with this PR. It looks like using the right context for the container is the solution. Let's make sure to share artifacts with the customers on #9103 to make sure that it's not going to be a problem.

Reply:
Great news! It looks like this PR actually resolves #9103.